platform/app/src/routes/buildModeRoutes.tsx

```diff
@@ -45,21 +46,23 @@ export default function buildModeRoutes({
       const path = `/${mode.routeName}/${dataSourceName}`;
 
       // TODO move up.
```

Comment:
To avoid duplicate code on line 75, please consider performing this TODO. I respect it if you were to decline, but I figure I would mention it.

Reply:
Updated, thanks!